pkg/virt-controller/services/template_test.go

```diff
@@ -17,25 +17,29 @@
  *
  */
 
-package services_test
+package services
```

Comment:
I find it helpful to target helper functions instead of building monstrosity tests that are not unit tests after all and that pull in lots of externals. (We need those too but not as the only testing layer.)